FoxHound.App/Blogs/CreateBlog/CreateBlogCommandValidator.cs

```diff
@@ -1,18 +1,13 @@
 ﻿using FluentValidation;
+using FoxHound.App.Blogs.Common;
 
 namespace FoxHound.App.Blogs.CreateBlog
 {
     public class CreateBlogCommandValidator : AbstractValidator<CreateBlogCommand>
     {
         public CreateBlogCommandValidator()
         {
-            RuleFor(x => x.Title)
-                .NotEmpty().WithMessage("Title is required")
-                .MaximumLength(128).WithMessage("Title must be less than or equal to {MaxLength} characters");
-
-            RuleFor(x => x.Owner)
-                .NotEmpty().WithMessage("Owner is required")
-                .MaximumLength(20).WithMessage("Owner must be less than or equal to {MaxLength} characters");
+            RuleFor(x => x).SetValidator(new CommonBlogCommandValidator());
```

Comment:
Consider adding validation rules here that when creating a blog that the owner must be unique (as a separate PR). This infrastructure now gives us a nice hook to keep all the common validation abstracted, but still allows to do specific validations as well. Good stuff here.

Reply:
sounds good, this will now be part of the CommonBlogValidator logic

Reply:
Yeah, i don't think the logic can be the same for both add and update. For add you'll want to check that the owner isn't in use on any blog. For update, you'll want to check that it isn't in use for any blog other than itself.